From 2ec264824788b726bf5ce6813ca5579cdb7cc5e4 Mon Sep 17 00:00:00 2001
From: David Pagan <dave.pagan@amd.com>
Date: Mon, 21 Aug 2023 16:19:59 -0700
Subject: [PATCH] Fix static_assert string literal that contains a "\%". This
 is no longer (#860)

valid. They can only be simple escape sequences. Removing '\' fixes
issue. Assert message now compiles and emits the '%' as expected.
---
 src/include/msccl/msccl_struct.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/include/msccl/msccl_struct.h b/src/include/msccl/msccl_struct.h
index ee03403a5..2e3204921 100644
--- a/src/include/msccl/msccl_struct.h
+++ b/src/include/msccl/msccl_struct.h
@@ -64,7 +64,7 @@ struct mscclThreadBlock {
   int16_t channelId; // associated channel. -1 indicates a thread block with only local copies
 }; // 5384 bytes
 
-static_assert(sizeof(struct mscclThreadBlock) % sizeof(uint64_t) == 0, "Sanity check: sizeof(struct mscclThreadBlock) \% sizeof(uint64_t) != 0");
+static_assert(sizeof(struct mscclThreadBlock) % sizeof(uint64_t) == 0, "Sanity check: sizeof(struct mscclThreadBlock) % sizeof(uint64_t) != 0");
 
 struct mscclFlag {
   uint64_t flag;
